libs/blocks/text/text.css

```diff
@@ -73,6 +73,25 @@
   margin: var(--spacing-m) 0;
 }
 
+.text-block .title-l {
```

Comment:
Where is this class name coming from? If possible we should try to align w/ the existing `-body` or `-detail` existing typography classes and not introduce more associations.

Reply:
@ryanmparrish yeah I copied this over from @meganthecoder's work on the Aside block -- Apparently we will have an update in the future that implements Spectrum 2 and renders this obsolete. For now, this is a band-aid and will require a due-diligence in the future :)

Reply:
Where is the example for this? I don't see why we need this when it sounds like tech-debt

Reply:
I'll let @meganthecoder speak to the why of this, but dfaf39fed9f10cd14986290cd1f96de75ea495a1 is where this got originally implemented on the Aside block. If we can reasonably rely on Spectrum 2 now, I'll happily remove this, but I have not heard otherwise.

Reply:
I mean in your [after link](https://ebartholomew-mwpw-156365-basic-blocks--milo--adobecom.hlx.page/drafts/ebartholomew/fragments/basic-blocks-text?martech=off). I don't see an example of this need/usage?

Reply:
lol, sorry about that, I handed that page off to authors for testing and it looks like they removed some things. I'll update the description with stable links 👍

Reply:
@ryanmparrish Title is a new stylistic element (like body and detail) that's going to be added in Spectrum 2. However, CC needs it now and can't wait until Spectrum 2 is officially out. The `title-l` class is our stopgap measure. As you can see, the large title is bold, 18px, and normal case, so none of the body or detail styles would work to match the requirements.

<img width="1450" alt="Screenshot 2024-09-03 at 3 50 53 PM" src="https://github.com/user-attachments/assets/158ef307-4aea-45e7-a391-33e89c55e5c6">

From: https://main--milo--adobecom.hlx.page/drafts/methomas/consonant-tiger/aside-standard-large?martech=off#authorable-text-sizes